x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_settings.tsx

```diff
@@ -0,0 +1,450 @@
+                  >
+                    {(list, search) => (
+                      <EuiPanel paddingSize="s" hasShadow={false}>
+                        {search}
+                        {list}
+                      </EuiPanel>
+                    )}
+                  </EuiSelectable>
+                ) : (
+                  <EuiComboBox
+                    options={indices.map(index => ({ label: index }))}
+                    placeholder={i18n.translate(
+                      'xpack.snapshotRestore.policyForm.stepSettings.indicesPatternPlaceholder',
+                      {
+                        defaultMessage: 'Enter index patterns, i.e. logstash-*',
+                      }
+                    )}
+                    selectedOptions={indexPatterns.map(pattern => ({ label: pattern }))}
+                    onCreateOption={(pattern: string) => {
+                      if (!pattern.trim().length) {
+                        return;
+                      }
+                      const newPatterns = [...indexPatterns, pattern];
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                    onChange={(patterns: Array<{ label: string }>) => {
+                      const newPatterns = patterns.map(({ label }) => label);
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                  />
+                )}
+              </EuiFormRow>
+            </Fragment>
+          )}
+        </Fragment>
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderIgnoreUnavailableField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescriptionTitle"
+              defaultMessage="Ignore unavailable indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescription"
+          defaultMessage="Ignores indices that are unavailable when taking the snapshot. Otherwise, the entire snapshot will fail."
+        />
+      }
+      idAria="policyIgnoreUnavailableDescription"
+      fullWidth
+    >
+      <EuiFormRow
+        hasEmptyLabelSpace
+        describedByIds={['policyIgnoreUnavailableDescription']}
+        fullWidth
+      >
+        <EuiSwitch
+          label={
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableLabel"
+              defaultMessage="Ignore unavailable indices"
+            />
+          }
+          checked={config.ignoreUnavailable}
```

Comment:
It is by default `undefined` instead of `false`. And gives the following error when toggled the first time

<img width="1005" alt="Screen Shot 2019-08-22 at 16 39 44" src="https://user-images.githubusercontent.com/2854616/63524071-9d83b280-c4fb-11e9-8182-758be1474265.png">